Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/ubi8/nodejs-10:latest
```

Comment:
Do you think we could rename this Dockerfile to indicate that it's for development only and not for production use? For instance, we can call it `Dockerfile-dev` or something.